onnxruntime/python/_pybind_state.py.in

```diff
@@ -8,16 +8,20 @@ Ensure that dependencies are available and then load the extension module.
 import os
 import platform
 import sys
+import warnings
 
 from . import _ld_preload  # noqa: F401
 
 if platform.system() == "Windows":
     from . import version_info
 
+    # If on Windows, check if this import error is caused by the user not installing the 2019 VC Runtime
+    # The VC Redist installer usually puts the VC Runtime dlls in the System32 folder, but it may also be found
+    # in some other locations or in some case Windows is not installed at C:\Windows.
+    # TODO, we may want to try to load the VC Runtime dlls instead of checking if the hardcoded file path
+    # is valid, and raise ImportError if the load fails
     if version_info.vs2019 and platform.architecture()[0] == "64bit":
         if not os.path.isfile("C:\\Windows\\System32\\vcruntime140_1.dll"):
-            raise ImportError(
-                "Microsoft Visual C++ Redistributable for Visual Studio 2019 not installed on the machine.")
-@ONNXRUNTIME_SETDLOPENFLAGS_GLOBAL@
```

Comment:
is `@ONNXRUNTIME_SETDLOPENFLAGS_GLOBAL@` needed?

Reply:
Accidentally removed